tools/deploy_assisted_installer_configmap.py

```diff
@@ -51,7 +51,7 @@ def get_deployment_tag(args):
 
 def main():
     utils.verify_build_directory(deploy_options.namespace)
-    verify_ocp_versions(json.loads(json.loads('"{}"'.format(deploy_options.ocp_versions))))
+    verify_ocp_versions(ocp_versions=None, release_images=json.loads(json.loads('"{}"'.format(deploy_options.release_images))))
```

Comment:
```suggestion
    verify_ocp_versions(ocp_versions=None, release_images=json.loads(json.loads(f'"{deploy_options.release_images}"')))
```
also, twice ``json.loads``?